mmaction/core/evaluation/eval_detection.py

```diff
@@ -147,6 +148,8 @@ def evaluate(self):
         return self.mAP, self.average_mAP
 
 
+# TODO: This function seems similar
+#  with `def average_precision_at_temporal_iou`
```

Comment:
@kennymckormick

Reply:
The two functions perform nearly the same operation and can be merged as one.

Reply:
I am not familiar with these two, could you help to merge ?

Reply:
OK.